pkg/cdi/container-edits.go

```diff
@@ -26,7 +26,7 @@ import (
 
 	oci "github.com/opencontainers/runtime-spec/specs-go"
 	ocigen "github.com/opencontainers/runtime-tools/generate"
-	"tags.cncf.io/container-device-interface/specs-go"
+	cdi "tags.cncf.io/container-device-interface/specs-go"
```

Comment:
Would it be better to import tags.cncf.io/container-device-interface/specs-go as is to avoid confusion with pkg/cdi?

Reply:
I thought about it. renaming it to cdispecs would trigger renaming oci -> ocispecs for consistency, so I'm still for letting it be specs, i.e. using not named import, unless it triggers linting failures.

Reply:
OK. `specs` is fine.

Reply:
> Would it be better to import tags.cncf.io/container-device-interface/specs-go as is to avoid confusion with pkg/cdi?

No, not in my opinion, since this is the `pkg/cdi` package itself so it never needs to reference itself `externally`. IMO/for me the most readable alternative is to reference the OCI Spec as `oci.Spec` and the CDI Spec as `cdi.Spec` in the code, with the next best option being `ocispec.Spec` and `cdispec.Spec` and that's why I chose to do it that way.

But if I in a minority with this opinion with some other consensus on what's best, just let me 
guys know what is the consensus and I'll update this PR accordingly.

Reply:
That makes sense to me.
/lgtm

if external project imports both the cdi and the specs-go packages they still can import them as is, i.e. as cdi and specs.